core/controllers/base.py

```diff
@@ -425,7 +462,13 @@ def _render_exception(self, error_code, values):
                 self.render_template('pages/error/error.html')
 
     def handle_exception(self, exception, unused_debug_mode):
-        """Overwrites the default exception handler."""
+        """Overwrites the default exception handler.
+
+        Args:
+            exception: The exception that was thrown.
+            unused_debug_mode: The uncaught exception.
```

Comment:
This is the [documentation note](https://webapp2.readthedocs.io/en/latest/api/webapp2.html#webapp2.WSGIApplication.handle_exception). The function `handle_exception` needs 3 arguments. And e reads - The uncaught exception. So, I am assuming this was just a psuedo name. Correct me if I am wrong and it has some other use